### PR TITLE
Update Intro to Sentry page

### PR DESCRIPTION
**Pull request by lizokm**

### CODE REVIEW

File: docs/product/index.mdx [hunk 1]
```diff
@@ -24,13 +24,13 @@ Sentry automatically captures unhandled exceptions (errors). It then groups simi
 
 ### Tracing
 
-Tracing gives you a connected view of your application from the frontend to the backend and between systems by looking at interactions among various components of your stack.
+Tracing gives you a full view of what's happening in your application from the frontend to the backend and between systems by looking at interactions between various components of your stack. This is especially useful if you rely on microservices or are trying to find the root cause of a difficult user complaint. With [distributed tracing](/concepts/key-terms/tracing/distributed-tracing/), you can quickly identify where performance bottlenecks are coming from.
```
Comment (lizokm) [inline docs/product/index.mdx]:
```suggestion
Tracing gives you a full view of what's happening in your application from the frontend to the backend and between systems, by looking at interactions between various components of your stack. This is especially useful if you rely on microservices or are trying to find the root cause of a difficult user complaint. With [distributed tracing](/concepts/key-terms/tracing/distributed-tracing/), you can quickly identify where performance bottlenecks are coming from.
```

File: docs/product/index.mdx [hunk 2]
```diff
@@ -24,13 +24,13 @@ Sentry automatically captures unhandled exceptions (errors). It then groups simi
...
+You can also get full context about what was happening when an error or performance issue occurred by looking at the [Trace View](/concepts/key-terms/tracing/trace-view/) page. This will allow you to among other things, trace slow-loading pages all the way back to poor-performing API calls. Lastly, you can use the [Trace Explorer](/product/explore/traces/) page to search span data, including conditions relating to the structure of the trace itself.
 
 ### Profiling
 
-Sentry [**Profiling**](/product/profiling/) provides code-level insights into your application performance in production so you can find and fix the problematic line of code faster. It profiles code in every environment your application is running on so you can quickly see which functions are contributing to a slow span.
+Sentry [**Profiling**](/product/profiling/) provides code-level insights into your application performance in production so you can find and fix the problematic lines of code faster. You can profile code in any environment your application is running on so you can quickly see which functions are causing a slow span.
```
Comment (coolguyzone) [inline docs/product/index.mdx:33]:
```suggestion
Sentry [**Profiling**](/product/profiling/) provides code-level insights into your application performance in production so you can find and fix problematic lines of code faster. You can profile code in any environment your application is running on so you can quickly see which functions are causing a slow span.
```

File: docs/product/index.mdx [hunk 3]
```diff
@@ -24,13 +24,13 @@ Sentry automatically captures unhandled exceptions (errors). It then groups simi
 
 ### Tracing
 
-Tracing gives you a connected view of your application from the frontend to the backend and between systems by looking at interactions among various components of your stack.
+Tracing gives you a full view of what's happening in your application from the frontend to the backend and between systems, by looking at interactions between various components of your stack. This is especially useful if you rely on microservices or are trying to find the root cause of a difficult user complaint. With [distributed tracing](/concepts/key-terms/tracing/distributed-tracing/), you can quickly identify where performance bottlenecks are coming from.
```
Comment (coolguyzone) [inline docs/product/index.mdx:27]:
```suggestion
Tracing gives you a full view of what's happening in your application from the frontend to the backend and between systems, by looking at interactions between various components of your stack. This is especially useful if you rely on microservices or are trying to find the root cause of a difficult user complaint. With [distributed tracing](/concepts/key-terms/tracing/distributed-tracing/), you can quickly identify performance bottlenecks.
```

**Review by coolguyzone — APPROVED** [review]
Added a couple of (opinionated) edits for brevity, other than that, lgtm!

**Comment by lizokm** [comment]
Thanks for the edits @coolguyzone!

**Pull request merged**